integration/kubernetes/runtimeclass_workloads/job.yaml

```diff
@@ -8,6 +8,7 @@ kind: Job
 metadata:
   name: job-pi-test
 spec:
+  ttlSecondsAfterFinished: 50
```

Comment:
Interesting idea. Questions...
 - I guess we figured out why pods were being left behind etc.? Can we not clean that up in the test itself? Or, is this a k8s delayed garbage collection thing??
 - why 50s - I would have thought we'd rather have more like 0s, so it got cleaned up immediately?
 - does the test have to wait for >50s somewhere then for the cleanup to actually happen? I didn't spot that in the code. I'm not sure I understand quite why/where/when this happens?